content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/log-in-chainguard-enforce/index.md

```diff
@@ -106,6 +106,42 @@ Valid! Id: <Relevant ID here>
 
 You are now logged in to Chainguard Enforce and can use `chainctl` for your account. 
 
+## Authenticating through another device with `chainctl`
+
+You can authenticate and log into Chainguard Enforce with any device (this will typically be a smartphone) when you cannot access your present working device's browser.
+
+To authenticate, you will browse to the provided URL on an alternate device.
+
+First, authenticate with `chainctl`.
+
+```sh
+chainctl auth login
+```
+
+You'll receive output that the device's browser could not open, and be provided with a URL to authenticate.
+
+```
+Authenticating...
+Error opening browser. Switching to device flow...
+
+Enter the verification code FTFD-ZVTV in your browser at: https://auth.chainguard.dev/activate
+Code will be valid for 900 seconds
+```
+
+When you navigate to https://auth.chainguard.dev/ with your smartphone or other device, you'll receive a screen with a field to enter the verification you were given in the terminal output.
```

Comment:
```suggestion
When you navigate to `https://auth.chainguard.dev/activate` with your smartphone or other device, you'll receive a screen with a field to enter the verification you were given in the terminal output.
```

Instead of backticks, would you want to make it a hyperlink?

Reply:
I thought this would automatically be a hyperlink, but it's not, so yeah I would want it to be the hyperlink

```suggestion
When you navigate to [https://auth.chainguard.dev/](https://auth.chainguard.dev/) with your smartphone or other device, you'll receive a screen with a field to enter the verification you were given in the terminal output.
```